input/data/2014/2014ApJ...785L..16A/tev-000067-1.yaml

```diff
@@ -0,0 +1,16 @@
+source_id: 67
+reference_id: 2014ApJ...785L..16A
+file_id: 1
+telescope: veritas
+
+data:
+  livetime: 28.5
+  significance: 8.5
+
+spec:
+  mjd: {min: 54881.0, max: 55003.0}
+  erange: {min: 0.12, max: 0.75, unit: TeV}
+  model:
+    type: pl
```

Comment:
You commented that you changed to `pl2`, but this change (e.g. for `input/data/2014/2014ApJ...785L..16A/tev-000067-1.yaml` isn't visible on Github yet. Did you commit and push?

---

Side comment: if you use `git pull --rebase` instead of `git pull` you'll avoid introducing merge commits here in your PRs, which overall make the version history more complex and hard to understand.